From: =?UTF-8?q?Ulrich=20=C3=96lmann?= <u.oelmann@pengutronix.de>
Date: Fri, 18 Sep 2026 11:26:34 +0200
Subject: [PATCH] include: ocotp.h: add missing include
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Otherwise GCC complains:

  <PATH>/barebox-2026.08.0/include/mach/imx/ocotp.h: In function 'imx_ocotp_read_uid':
  <PATH>/barebox-2026.08.0/include/mach/imx/ocotp.h:47:16: error: implicit declaration of function 'readl' [-Werror=implicit-function-declaration]
     47 |         uid  = readl(ocotp + OCOTP_UID_H);
        |                ^~~~~

Upstream-Status: Pending

Signed-off-by: Ulrich Ölmann <u.oelmann@pengutronix.de>
---
 include/mach/imx/ocotp.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/mach/imx/ocotp.h b/include/mach/imx/ocotp.h
index e6f62a9da48c..2d8fcc7c2657 100644
--- a/include/mach/imx/ocotp.h
+++ b/include/mach/imx/ocotp.h
@@ -5,6 +5,7 @@
 
 #include <linux/bitfield.h>
 #include <linux/types.h>
+#include <linux/io.h>
 
 #define OCOTP_SHADOW_OFFSET		0x400
 #define OCOTP_SHADOW_SPACING		0x10
